engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -375,6 +380,7 @@ public static boolean addAppender(String name, Properties properties)
       try {
         Method method = PropertyConfigurator.class.getDeclaredMethod("parseAppender", Properties.class, String.class);
         method.setAccessible(true);
+        makeMDCProperties();
```

Comment:
MDC properties should be available for all appenders, including statically defined appenders